### PR TITLE
fix naming of Rideshare for consistency

### PR DESCRIPTION
**Pull request by evelynhasama**
**Description:**
changed naming across website to Rideshare to maintain consistency

**Test:**
tested in local environment

### CODE REVIEW

File: app/controllers/rides_controller.rb [hunk 1]
```diff
@@ -211,47 +211,47 @@ def airport_helper(type)
...
+      :title => "5C Rideshare :: to ontario",
       :airport => "To Ontario",
       :ftype => "Departure",
       :rides => Ride.where(airport: "To Ontario").all,
       :check => true
       }
     when "2"
       {
-      :title => "5C ride share :: from ontario",
+      :title => "5C Rideshare :: from ontario",
```
Comment (summerhasama) [inline app/controllers/rides_controller.rb:222]: Caps Ontario

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ class UsersController < ApplicationController
   before_filter :correct_user, :only => [:edit, :show]
 
   def edit
-    @title = "5C ride share :: settings"
+    @title = "5C Rideshare :: settings"
```
Comment (summerhasama) [inline app/controllers/users_controller.rb:5]: Caps Settings

File: app/assets/javascripts/rides.js.erb [hunk 3]
```diff
@@ -34,25 +34,25 @@ $(document).ready(function() {
...
-		case "5C ride share :: from LAX":
+		case "5C Rideshare :: from LAX":
 			nav.eq(3).addClass('current-page')
 			break
-		case "5C ride share :: to ontario":
+		case "5C Rideshare :: to ontario":
 			nav.first().addClass('current-page')
 			break
-		case "5C ride share :: from ontario":
+		case "5C Rideshare :: from ontario":
```
Comment (summerhasama) [inline app/assets/javascripts/rides.js.erb:46]: Caps Ontario

File: app/assets/javascripts/rides.js.erb [hunk 4]
```diff
@@ -34,25 +34,25 @@ $(document).ready(function() {
...
 			nav.eq(1).addClass('current-page')
 			break
-		case "5C ride share :: new":
+		case "5C Rideshare :: new":
```
Comment (summerhasama) [inline app/assets/javascripts/rides.js.erb:49]: Caps new

File: app/assets/javascripts/rides.js.erb [hunk 5]
```diff
@@ -34,25 +34,25 @@ $(document).ready(function() {
...
 			$("#create-new-ride").children().eq(1).addClass('current-page')
 			break
-		case "5C ride share :: settings":
+		case "5C Rideshare :: settings":
 			$("#create-new-ride").children().eq(0).addClass('current-page')
 			break
-		case "5C ride share :: your rides":
+		case "5C Rideshare :: your rides":
```
Comment (summerhasama) [inline app/assets/javascripts/rides.js.erb:55]: Caps

File: app/assets/javascripts/rides.js.erb [hunk 6]
```diff
@@ -34,25 +34,25 @@ $(document).ready(function() {
...
-		case "5C ride share :: from ontario":
+		case "5C Rideshare :: from ontario":
 			nav.eq(1).addClass('current-page')
 			break
-		case "5C ride share :: new":
+		case "5C Rideshare :: new":
 			$("#create-new-ride").children().eq(1).addClass('current-page')
 			break
-		case "5C ride share :: settings":
+		case "5C Rideshare :: settings":
```
Comment (summerhasama) [inline app/assets/javascripts/rides.js.erb:52]: Caps \

**Review by summerhasama — CHANGES_REQUESTED** [review]
Please fix all titles to have title casing

**Comment by evelynhasama** [comment]
hi @summerhasama I believe Liz is on that task for fixing the rest of the web tab header names (along with rendering the logo).

**Review by summerhasama — APPROVED** [review]

**Pull request merged**